### DISCUSSION TITLE
How to use custom font on Windows?

### DISCUSSION THREAD

**Discussion by LauraLaureus** [ImageMagick/ImageMagick#1985] [General]
Hi!!
I have been trying to use custom fonts on Windows but magick doesn't list it as available. 
I tried my editing my own type-windows.xml and adding it to the type.xml without any luck. 

Is there any tutorial specific for Windows I have miss? All I have found is for linux distribution (including the type-windows.xml that is available has Linux paths >.<)

Thanks in advantage.

**Comment by snibgo** [comment]
What version of IM? What version of Windows? What do you mean by "custom" fonts?

If you install a font in Windows, it should be available to IM. Use "magick -list font" to see if it is available.

If your font is in a ".ttf" file, you don't need to install it in windows. Just use the "-font" option with the filename (and path if not in the current directory):
`magick -font annifont.ttf label:"Hello world" x.png`

**Reply by LauraLaureus** [reply]
I'm using latest available on the web. 
I have downloaded http://fontsgeek.com/fonts/Gotham-Black-Regular and installed on Windows but it's not listed. 

I prefer to be detected by magick in order to avoid non-dev users to look for the path.
Can you help me @snibgo ?
